content/en/_index.html

```diff
@@ -40,6 +40,22 @@ <h3 class="section-head">
   </div>
 </div>
 
+<div id="overview" class="text-center">
```

Comment:
The second element under the outer div should probably be another div with the `container` class, so that everything is contained within the pages padding.

You might need to remove the `container` class from the logos div also.